### ISSUE TITLE
Update fit$loglik

### ISSUE THREAD

**Issue by pcarbo** [stephenslab/udr#17]
Currently `fit$loglik` is not updated. For example I see this:

```r
> fit.ed$loglik
[1] -135146.1
> max(fit.ed$progress$loglik)
[1] -81304.91
```

**Comment by yunqiyang0215** [comment]
Fixed, see commit 34919b7e05924149933ab15c2f86381d12a41875

**Comment by pcarbo** [comment]
Great, thanks!
